src/components/Header.tsx

```diff
@@ -39,25 +53,48 @@ const Header = (props: BoxProps) => {
                 textDecorationColor: 'primary.900',
               }}
             >
-              <HStack spacing={2}>
+              <HStack spacing={{ base: 1, lg: 2 }}>
                 <Box h={LOGO_SIZES} w={LOGO_SIZES}>
-                  <Image src={mascotIcon} alt="" height={150} width={150} />
+                  <Image
+                    src={mascotIcon}
+                    alt=""
+                    height={70}
+                    width={70}
+                    aria-hidden
+                  />
                 </Box>
                 <Heading
                   as="h1"
                   color="primaryText"
                   letterSpacing="tight"
                   fontWeight="black"
-                  fontSize={{ base: '16px', sm: '36px' }}
+                  fontSize={{ base: '16px', md: '36px' }}
                 >
                   Octoclairvoyant
                 </Heading>
               </HStack>
             </RouteLink>
           </Flex>
-          {!!process.env.NEXT_PUBLIC_FEATURE_FLAG_COLOR_MODE && (
-            <ToggleColorModeButton />
-          )}
+          <HStack spacing={{ base: 4, md: 10 }}>
+            {!!process.env.NEXT_PUBLIC_FEATURE_FLAG_COLOR_MODE && (
+              <ToggleColorModeButton
+                boxSize={LOGO_SIZES}
+                minWidth={LOGO_SIZES}
+              />
+            )}
+            <Link
+              href={REPO_URL}
+              aria-label="Octoclairvoyant repository on GitHub"
+              isExternal
+            >
+              <Icon
```

Comment:
The focus ring for the GitHub icon is square. Can we make it round so it follows the _on hover_ outline?

![github-icon-square-focus-ring](https://user-images.githubusercontent.com/34918129/153856316-2e876ea6-6a85-4d44-b348-63d432b760c6.png)

Reply:
Good catch! It should be definitely possible since the previous button has a full rounded focus ring.

Reply:
Is `rounded: 'full'` what we need to get the full rounded focus ring?

Example from the same file on line 94:

```
               _hover={{ rounded: 'full', bgColor: 'monochromeInverted4' }}
```